.travis.yml

```diff
@@ -16,7 +16,7 @@ script:
   - 'bundle exec rake $CHECK'
 bundler_args: --without system_tests release
 rvm:
-  - 2.5.7
+  - 2.5.8
```

Comment:
I'd prefer to just use `2.5`. IMHO there's no real point in pinning to a .z nowadays. Ruby is stable enough that it should just work. Same goes below for 2.4 instead of 2.4.9.